package.json

```diff
@@ -9,9 +9,7 @@
     }
   },
   "lint-staged": {
-    "src/**/*.{js,jsx,ts,tsx}": [
```

Comment:
not sure why changes to this file happened?

Reply:
My editor decided to format it that way. Going to return to normal.